static/src/js/components/KeywordTreeContextMenu/__tests__/KeywordTreeContextMenu.test.jsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react'
+import {
+  render,
+  fireEvent,
+  screen
+} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import {
+  describe,
+  test,
+  expect,
+  vi
+} from 'vitest'
+import { KeywordTreeContextMenu } from '../KeywordTreeContextMenu'
+
+describe('KeywordTreeContextMenu component', () => {
+  const mockOptions = [
+    {
+      id: 'edit',
+      label: 'Edit',
+      action: vi.fn()
+    },
+    {
+      id: 'delete',
+      label: 'Delete',
+      action: vi.fn()
+    }
+  ]
+
+  const mockOnClose = vi.fn()
+  const mockRef = { current: null }
+
+  const renderComponent = () => render(
+    <KeywordTreeContextMenu
+      x={100}
+      y={100}
+      onClose={mockOnClose}
+      options={mockOptions}
+      forwardedRef={mockRef}
+    />
+  )
+
+  describe('when Rendering and Positioning', () => {
```

Comment:
when rendering and positioning context menu

Reply:
Done